libsrc/c64/soft80_bordercolor.s

```diff
@@ -0,0 +1,17 @@
+;
+; Groepaz/Hitmen, 12.10.2015
+;
+; unsigned char __fastcall__ bordercolor (unsigned char color);
+;
+
+        .export         soft80_bordercolor
+
+        .include        "c64.inc"
+
+        ; FIXME: if we'd move this function into a seperate file in the regular
+        ;        conio lib, then we dont need this override at all.
+soft80_bordercolor:
```

Comment:
Soft80 is the inspiration for separating `bordercolor()` from the other color functions.  Therefore, I believe that you should do it in this Pull Request.